modules/Material/Slider.qml

```diff
@@ -0,0 +1,225 @@
+/*
+ * QML Material - An application framework implementing Material Design.
+ * Copyright (C) 2014 Jordan Neidlinger
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.0
+import QtQuick.Controls 1.2 as Controls
+import QtQuick.Controls.Styles 1.2 as ControlStyles
+import Material 0.1 as Material
+
+Controls.Slider {
+    id: slider
+
+    /*!
+       Set to \c true to enable a float numeric value label above the slider knob
+     */
+    property bool numericValueLabel: false
+
+    /*!
+       Set to \c true if the switch is on a dark background
+     */
+    property bool darkBackground
+
+    tickmarksEnabled: false
+    implicitHeight: numericValueLabel ? Material.units.dp(54) : Material.units.dp(32)
+    implicitWidth: Material.units.dp(200)
+    style: ControlStyles.SliderStyle {
+        property Component knob : Item {
+            implicitHeight: control.pressed || control.activeFocus ? Material.units.dp(32) : 0
+            implicitWidth: control.pressed || control.activeFocus ? Material.units.dp(32) : 0
+
+            Material.Label {
+                anchors.fill: parent
+                horizontalAlignment: Qt.AlignHCenter
+                verticalAlignment: Qt.AlignVCenter
+                text: control.value
+                color: Material.Theme.lightDark(Material.Theme.primaryColor,
```

Comment:
Isn't this note:

Light

Thumb on: Swatch 500, Alpha 100%
Track on: Swatch 500, Alpha 100%

and 

Dark

Thumb on: Swatch 200, Alpha 100%
Track on: Swatch 200, Alpha 100%

"Swatch 500" are the primary color swatches. The spec isn't very well documented though.